pkg/mpc-examples/contracts/interfaces/IManagedPoolFactory.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+
+import "@balancer-labs/v2-interfaces/contracts/solidity-utils/openzeppelin/IERC20.sol";
+
+interface IManagedPoolFactory {
```

Comment:
Can we request an interface from the SC Team instead of copying this here?

Reply:
IIRC, this was not an interface copied from the monorepo, but rather something we (I) put together. I'm definitely in favor of this being something we import though.

One detail is that this struct has changed name and contents [before](https://github.com/balancer/balancer-v2-monorepo/blob/9d2ca0cfacb704a4fbf639ba7d7b9bcf88572ff8/pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol#L94-L105) deprecation and [after](https://github.com/balancer/balancer-v2-monorepo/blob/master/pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol#L94-L102). Do you think they would be amenable to putting together and packaging an interface for something they no longer support, or would it be more likely that we would just sit with this until they deploy MPv2 with interfaces for that?

For the other two fns, `create` was taken directly from `ManagedPoolFactory.sol`, not an interface, while `getLastCreatedPool` was taken from some of the LinearPoolFactory samples. 

For the latter, we can/should just import `pool-utils/ILastCreatedPoolFactory.sol`.

Reply:
Alright, let's continue using this until we get a new `ManagedPoolFactory` deployment, at which time we can also request an `IManagedPoolFactory` interface.